Update lightCurve.py

leightontanner opened the pull request:
Updated the lightCurve plotting function to be more general to work with a more general data set; in particular, the previous version assumes the oid changes at particular locations in the list, and the software of this new version determines that. This version still assumes there are at least two unique object id's and no more than three.

leightontanner committed:
Update lightCurve.py

Updated the lightCurve plotting function to be more general to work with a more general data set; in particular, the previous version assumes the oid changes at particular locations in the list, and the software of this new version determines that. This version still assumes there are at least two unique object id's and no more than three.

adamamiller reviewed (changes requested):
There are sources with only one object id and others with >3, I will merge with the repo when a more general version of the software is available.

leightontanner committed:
Update lightCurve.py

No major software improvements - just better commenting. I want to update this most recent version before I rework the code to make it sufficiently general to handle sources with only one object id and others with >3.

leightontanner committed:
Update lightCurve.py

Suitably generalized to deal with sources with any number of object ID's!

leightontanner committed:
Update lightCurve.py

There are sources with only one object id and others with >3, and Adam said he will merge with the repo when a more general version of the software is available. This version of the lightCurve software is the fully general version; it works for sources with any number of object ID's!

leightontanner committed:
Update lightCurve.py

Upgraded the software to produce phase folded light curves.

leightontanner committed:
Update lightCurve.py

works incorrectly but updated to only load .tbl

leightontanner committed:
Update lightCurve.py

Plotted two periods of the light curve to get better feel for its character.

leightontanner committed:
Update lightCurve.py

Updated commenting to fully general version of light curve plotting software.

leightontanner committed:
Update lightCurve.py

Functions updated to contain docstrings!
